examples/tokenvm/DEVNETS.md

```diff
@@ -12,17 +12,17 @@ page](https://github.com/ava-labs/avalanche-ops/releases/tag/latest).
 #### Option 1: Install `avalanche-ops` on Mac
 ```bash
 rm -f /tmp/avalancheup-aws
-wget "https://github.com/ava-labs/avalanche-ops/releases/download/latest/avalancheup-aws.aarch64-apple-darwin"
```

Comment:
I prefer to use aarch64 (for new M-series)

Reply:
That being said, I think M-series can still run x86_64 using emulation.

Reply:
ok I think that's correct emulators can be used, anyway will go back on this. I do also prefer to use aarch64 :D